docusaurus-docs/community/contribute/local-setup-ui.md

```diff
@@ -9,46 +9,62 @@ To get started with conda-store-ui development, there are a couple of options. T
-3. You can then log in using the default username of `username` and default password of `password`.
+1. [Fork the conda-store-ui repository](https://github.com/conda-incubator/conda-store-ui/fork)
+2. `git clone` the fork to your computer.
 
-**Note:** Hot reloading is enabled, so when you make changes to source files, your browser will reload and reflect the changes.
+## Basic Option (recommended): Run the UI in Docker
+
+For basic local development on the UI, running conda-store-ui in Docker is the simplest way to get started.
+
+1. Change your working directory to the project root: `cd conda-store-ui`
+2. Optional. Set environment variables. Copy `.env.example` to `.env`. All default settings should work, but if you want to test against a different version of conda-store-server, you can specify it in the `.env` file by setting the `CONDA_STORE_SERVER_VERSION` variable to the desired version. If you skip this step, it will be done automatically for you.
+3. Run `docker compose --profile local-dev up --build` to start the entire development stack. This step may take a few minutes the first time you run it.
+4. Open your local browser and go to [http://localhost:8000](http://localhost:8000) to see conda-store-ui.
+5. You can then log in using the default username of `username` and default password of `password`.
+
+**Note:** Hot reloading is enabled, so when you make changes to source files (i.e., files under the conda-store-ui/src/ directory), your browser will reload and reflect the changes.
 
-## Without Docker
+## Advanced Option: Run the UI locally (with the rest of the stack in Docker)
 
-This setup still uses Docker for supporting services but runs conda-store-ui locally.
+For more advanced development on conda-store-ui, the first option might not be sufficient. If you need to work extensively in the UI codebase, then you will probably want to run the UI web app locally rather than within a Docker container.
 
-### Set up your environment
+Note: this setup still uses Docker to run the rest of the Conda Store stack. That means that the Conda Store database, server, worker, and storage services will all run in Docker containers. But the frontend web app (conda-store-ui) will run locally (not in a Docker container) with this setup.
+
+### Set up your Conda environment
 
 This project uses [Conda](https://conda.io) for package management. To set up Conda, please see their [installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
-1. Change to the project root ` cd conda-store-ui`
-2. From the project root create the conda environment `conda env create -f environment_dev.yml`
-3. Activate the development environment `conda activate cs-ui-dev-env`
-4. Install yarn dependencies `yarn install`
 
-### Run the application
+1. Change to the project root: `cd conda-store-ui`
+2. From the project root, create the conda environment: `conda env create -f environment_dev.yml`
```

Comment:
```suggestion
2. From the project root, create the conda environment: `conda env create -f conda-store-server/environment_dev.yml`
```
Right??

Reply:
No, the original wording is the correct one since this belongs to `conda-store-ui`

Reply:
ohhh. Dang its so hard to keep straight.